contracts/common/Initializable.sol

```diff
@@ -2,28 +2,17 @@ pragma solidity ^0.4.18;
 
 
 contract Initializable {
-    uint256 private initializationBlock;
+    uint256 public initializationBlock;
```

Comment:
We keep it private since there is no need for Apps to read that value or modify it